lasagne/tests/layers/test_normalization.py

```diff
@@ -50,8 +50,8 @@ def ground_truth_normalizer(c01b, k, n, alpha, beta):
         for c in range(out.shape[2]):
             for x in range(out.shape[3]):
                 out[:, r, c, x] = ground_truth_normalize_row(
-                        row=c01b[:, r, c, x],
-                        k=k, n=n, alpha=alpha, beta=beta)
+                    row=c01b[:, r, c, x],
+                    k=k, n=n, alpha=alpha, beta=beta)
```

Comment:
Please don't do random code style changes as part of a PR. I know some editors are set to do that, but please don't commit such changes. The previous version was better anyway, because the extra indentation level makes clear it's not the next level of control.